src/montecarlo/run_scc_mcs.jl

```diff
@@ -183,7 +183,7 @@ function run_scc_mcs(model::model_choice;
 
     # Run the simulation
     sim_results = run(mcs, models, trials;
-        results_output_dir = joinpath(output_dir, "mcs_data"), 
+        results_output_dir = joinpath(output_dir, "scc_mcs_model_data"), 
```

Comment:
@lrennels is there a flag we can add somewhere such that when `save_list` is empty (or not specified at all in the `run_scc_mcs()`) that this directory doesn't get created? Right now it creates the directory, it is just empty.

Reply:
Ah interesting yea, that's like if you run the regular Monte Carlo without setting `output_dir` to `nothing` you get a automatically created empty folder right? Then I'm guessing here you'd end up with an error because it would try to join `nothing` with `"scc_mcs_model_data"` ... we should be able to just add an if-then statement on line 188 or something ...

Reply:
@bryanparthum I think this should do it https://github.com/rffscghg/MimiIWG.jl/pull/63

Reply:
brilliant.